ts/main.ts

```diff
@@ -156,6 +189,36 @@ function clearCardList(): void {
   }
 }
 
+function populateExerciseDetails(baseId: number): void {
+  for (const exercise of exerciseObjArr) {
+    if (exercise.baseId === baseId) {
+      $detailsTitle.innerHTML =
+        exercise.name +
+        ' <i class="fa-regular fa-heart" style="color: #ffc300"></i>';
```

Comment:
You should avoid using `innerHTML`. This can cause security issues because someone could theoretically find a way to inject malicious code into it.